auth_otp/services/auth_service.py

```
@@ -0,0 +1,437 @@
+                            timedelta(minutes=5)
+                        ).total_seconds()
+                    )
+                }
+                
+        except TokenError as e:
+            logger.error(f"Token refresh error: {e}")
+            return False, {
+                'error': 'invalid_token',
+                'message': 'توکن نامعتبر است'
+            }
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)